File: mapsms/twilio/send-sms.js
// Twilio Credentials 
var accountSid = 'AC0c0ef57b1f1d72fb7459217fe973acc5'; 
var authToken = 'e7d5395ff8430ecaa90e9156f47dea30'; 
 
//require the Twilio module and create a REST client 
var client = require('twilio')(accountSid, authToken); 
 
client.messages.create({ 
    to: "+17818798963", 
    from: "+15169864627", 
    body: "This is the ship that made the Kessel Run in fourteen parsecs?", 
}, function(err, message) { 
	if (err) {
		console.log(err.message);
	} else {
    	console.log(message.sid); 
	}
});

File: send_sms.py
from twilio.rest import TwilioRestClient

def send_responce(client, number = "+17742185184", txt = "Default message"):
	message = client.messages.create(to=number, from_="+15169864627", body=txt)
	#client.messages.delete(message.sid)




account_sid = 'AC0c0ef57b1f1d72fb7459217fe973acc5'; 
auth_token = 'e7d5395ff8430ecaa90e9156f47dea30'; 
client = TwilioRestClient(account_sid, auth_token)
last_sid = client.messages.list()[0].sid

while 1:
	if len(client.messages.list(to = "+15169864627")):
		msg = client.messages.list(to = "+15169864627")[0]
		print(msg.body)
		client.messages.delete(msg.sid)
	else:
		print("no messages found")
